src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import { TabsProps, Tab } from '../../types';
+
+export const Tabs: React.FC<TabsProps> = ({
+  selectedTabId,
+  tabs,
+  onTabSelected,
+}) => {
+  const handleClick = (tab: Tab) => {
+    if (selectedTabId !== tab.id) {
+      onTabSelected(tab);
+    }
+  };
+
+  const selectedTab = tabs.find((tab) => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => (
+            <li
+              key={tab.id}
+              data-cy="Tab"
+              className={selectedTab.id === tab.id ? 'is-active' : ''}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => handleClick(tab)}
+              >
+                {tab.title}
+              </a>
+            </li>
+          ))}
+        </ul>
+      </div>
+
+      <div className="block" data-cy="TabContent">
+        {tabs.find((tab) => tab.id === selectedTabId)?.content}
```

Comment:
I think you can use here selectedTab that you defined on line 14.

Reply:
I agree

Reply:
I agree too! 😀 Done